build.gradle

```diff
@@ -186,16 +186,12 @@ test {
 tasks.withType(Test).all {
 	// suppress all console output during testing unless running `gradle -i`
 	logging.captureStandardOutput(LogLevel.INFO)
-	maxHeapSize = "1024m"
-	if (name == 'springIoJdk8Test') {
-		exclude '**/*KafkaTests*'
-	}
-
```

Comment:
Since we have move to Kafka 0.10 there is no more reason to exclude Kafka 0.8 tests from the IO build. There is just no Kafka 0.8 tests and we are fully compatible with IO Brussels!